ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml

```diff
@@ -218,20 +205,6 @@ tests:
     - ref: openstack-test-openstack
     workflow: openshift-e2e-openstack-ipi
   timeout: 7h0m0s
-- as: openstack-kuryr-migration
```

Comment:
The only place we run kuryr-migration is 4.14, I wonder if it's not worth to keep running it, but on the nfv cloud.

If we think it's fine to stop running it, [the workflow code](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/openstack/kuryr-migration/openshift-e2e-openstack-kuryr-migration-workflow.yaml) needs to be removed.

Reply:
@gryf @mandre @dulek Thoughts?

Reply:
This job has been failing for a while
https://prow.ci.openshift.org/job-history/origin-ci-test/logs/periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

Reply:
Ah, better to keep that probably, it's our escape-route from Kuryr, it should work.

Reply:
I think we can remove it. I've added a second commit to remove the kuryr-migration workflow and step.